maintenance_team_hierarchy/tests/test_team_hierarchy.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase, tagged
+
+
+# This test should only be executed after all modules have been installed.
+@tagged("post_install", "-at_install")
+class TestTeamHierarchy(TransactionCase):
+    def setUp(self):
```

Comment:
And also replace `setUp` usage by `setUpClass`.

Reply:
Thanks for the corrections, I made the adjustments

Reply:
Thanks for the feedback, I made the adjustments regarding what was mentioned for changes